frontend/src/libs/packages/store/store.package.ts

```diff
@@ -11,11 +11,13 @@ import { notification } from '#libs/packages/notification/notification.js';
 import { handleError } from '#libs/packages/store/middlewares/middlewares.js';
 import { authApi } from '#packages/auth/auth.js';
 import { reducer as authReducer } from '#slices/auth/auth.js';
+import { chatReducer } from '#slices/chat/chat.js';
```

Comment:
We already get used to exporting all reducers as `reducer` and write `as` in imports, like in this file. It's better to use the same approach